shared/util/quit-helper.desktop.js

```diff
@@ -57,6 +67,7 @@ export function executeActions (actions: Array<Action>) {
 
 export function setupExecuteActionsListener () {
   ipcMain.on('executeActions', (event, actions) => {
-    executeActions(actions)
+    console.log('executeActionsRecieved', actions)
```

Comment:
just to help us debug, if we don't do this its kinda hard to tell whats going on in the rpc (esp with log send)